src/test/java/uk/nhs/hee/tis/trainee/credentials/service/CredemtoalEvemtServiceTest.java

```diff
@@ -0,0 +1,176 @@
+  private static final String PROGRAMME_2_traineeId = "47165";
+  private static final String PROGRAMME_2_tisId = "72055636-1439-11ee-be56-0242ac120002";
+  private static final Instant PROGRAMME_2_issuedAt =
+      Instant.parse("2023-08-27T09:41:35.000+00:00");
+
+  CredentialMetadata placement1 = new CredentialMetadata();
+  CredentialMetadata placement2 = new CredentialMetadata();
+  CredentialMetadata placement3 = new CredentialMetadata();
+  CredentialMetadata programme1 = new CredentialMetadata();
+  CredentialMetadata programme2 = new CredentialMetadata();
+
+  private CredentialEventService credentialEventService;
+
+  @Mock
+  CredentialMetadataRepository repository;
+
+  @BeforeEach
+  public void setUp() {
+    repository = mock(CredentialMetadataRepository.class);
+    credentialEventService = new CredentialEventService(repository);
+
+    placement1.setIssuedAt(PLACEMENT_1_issuedAt);
+    placement1.setTisId(PLACEMENT_1_tisId);
+    placement1.setCredentialId(String.valueOf(PLACEMENT_1_credentialId));
+    placement1.setTraineeId(PLACEMENT_1_traineeId);
+    placement1.setCredentialType("TRAINING_PLACEMENT");
+
+    placement2.setIssuedAt(PLACEMENT_2_issuedAt);
+    placement2.setTisId(PLACEMENT_2_tisId);
+    placement2.setCredentialId(String.valueOf(PLACEMENT_2_credentialId));
+    placement2.setTraineeId(PLACEMENT_2_traineeId);
+    placement2.setCredentialType("TRAINING_PLACEMENT");
+
+    placement3.setIssuedAt(PLACEMENT_3_issuedAt);
+    placement3.setTisId(PLACEMENT_3_tisId);
+    placement3.setCredentialId(String.valueOf(PLACEMENT_3_credentialId));
+    placement3.setTraineeId(PLACEMENT_3_traineeId);
+    placement3.setCredentialType("TRAINING_PLACEMENT");
+
+    programme1.setIssuedAt(PROGRAMME_1_issuedAt);
+    programme1.setTisId(PROGRAMME_1_tisId);
+    programme1.setCredentialId(String.valueOf(PROGRAMME_1_credentialId));
+    programme1.setTraineeId(PROGRAMME_1_traineeId);
+    programme1.setCredentialType("TRAINING_PROGRAMME");
+
+    programme2.setIssuedAt(PROGRAMME_2_issuedAt);
+    programme2.setTisId(PROGRAMME_2_tisId);
+    programme2.setCredentialId(String.valueOf(PROGRAMME_2_credentialId));
+    programme2.setTraineeId(PROGRAMME_2_traineeId);
+    programme2.setCredentialType("TRAINING_PROGRAMME");
+  }
+
+  @Test
+  void getTheLatestPlacementCredential() {
+    CredentialType type = CredentialType.valueOf("TRAINING_PLACEMENT");
+    String token = TestJwtUtil.generateTokenForTisId("47165");
+
+    List<CredentialMetadata> allPlacementCredentialList = new ArrayList<>();
+    allPlacementCredentialList.add(placement1);
+    allPlacementCredentialList.add(placement2);
+    allPlacementCredentialList.add(placement3);
+
+    List<CredentialMetadata> filteredPlacementList = new ArrayList<>();
+    filteredPlacementList.add(placement1);
+    filteredPlacementList.add(placement3);
+
+    when(repository.findByCredentialTypeAndTraineeId(type.getIssuanceScope(), token))
+        .thenReturn(allPlacementCredentialList);
```

Comment:
We can clean the initialization up by using `...thenReturn(List.of(placement1, placement2, placement3))`